client/src/components/Forms.js

```diff
@@ -49,18 +48,29 @@ const SignUpForm = (props) => {
             <Label htmlFor="signup-password">Password</Label>
             <Input type="password" name="signupPassword" id="signupPassword" onChange={props.onChange} value={props.passwordValue} placeholder="Password" />
           </FormGroup>
+          
+          <FormGroup check className="form-check-inline">
+              <Label check>
+                <Input  onChange={props.onChange} type="checkbox" name="terms" value="1" />{' '}
+                By checking this box, I agree to Train Method's terms and conditions
+                <br />
+              </Label>
+            </FormGroup>
+          
         </Form>
-        <Button onClick={props.onClick}>Sign Up</Button>
-      </CardBody> 
+        <br />
+        <button type="button" class="btn btn-dark" onClick={props.onClick}>Sign Up</button>
+        <br />
+      
     </div>
   );
 }
  
 
 const LogInForm = (props) => {
   return ( 
-    <div>
-      <CardBody id="login-body">
+    <div className="card-body" id="login-body">
```

Comment:
card-body without a card surrounding it, it may be bringing in a lot of style stuff its not supposed to